quantumdb-postgresql/src/test/java/io/quantumdb/core/backends/integration/videostores/MakeStoreFieldInStaffTableNullable.java

```diff
@@ -24,10 +26,9 @@
 import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Test;
 
-@Ignore // TODO: There seems to be something wrong with this test. Must investigate further.
+// @Ignore - Seems to be working again
```

Comment:
If so, feel free to drop this line and reenable the test. Thanks for checking 👍

Reply:
It worked when I added my extra columns, right now it breaks again... @Ignore once again